apps/todos/src/app/modules/add-edit-todo-form/add-edit-todo-form.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Input, OnInit, AfterViewInit, TemplateRef, ViewChild, Output, EventEmitter } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+
+import { AddTodo } from '@learning-workspace/api-interfaces';
+
+@Component({
+  selector: 'learning-workspace-add-edit-todo-form',
+  templateUrl: './add-edit-todo-form.component.html',
+  styleUrls: ['./add-edit-todo-form.component.scss'],
+})
+export class AddEditTodoFormComponent implements OnInit, AfterViewInit {
+  public todoForm: FormGroup;
```

Comment:
please use this property below Input() and Output()

Reply:
did it